manifests/a/Amazon/EC2Launch/2.0.2046.0/Amazon.EC2Launch.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Amazon.EC2Launch
+PackageVersion: 2.0.2046.0
+PackageLocale: en-US
+Publisher: Amazon

```

```suggestion
Publisher: Amazon Web Services
```

To match what's shown on Control Panel / Apps&Features